### ISSUE TITLE
[LSP]        Failed to handle notification textDocument/documentSymbol

### ISSUE THREAD

**Issue by mickaelistria** [OmniSharp/omnisharp-roslyn#1083]
I get the following stacktrace when running Omnisharp LSP support in Eclipse IDE

```
        Failed to handle notification textDocument/documentSymbol
System.NullReferenceException: Object reference not set to an instance of an object
  at OmniSharp.LanguageServerProtocol.Helpers.ToUri (System.String fileName) [0x00000] in <7b5a7a76b82241e79b11d53ab5acb828>:0 
  at OmniSharp.LanguageServerProtocol.Handlers.DocumentSymbolHandler.ToDocumentSymbol (OmniSharp.Models.MembersTree.FileMemberElement node, System.Collections.Generic.List`1[T] symbolInformationContainer, System.String containerName) [0x0003e] in <7b5a7a76b82241e79b11d53ab5acb828>:0 
  at OmniSharp.LanguageServerProtocol.Handlers.DocumentSymbolHandler+<Handle>d__6.MoveNext () [0x000b2] in <7b5a7a76b82241e79b11d53ab5acb828>:0 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <c9f8153c41de4f8cbafd0e32f9bf6b28>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x0003e] in <c9f8153c41de4f8cbafd0e32f9bf6b28>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in <c9f8153c41de4f8cbafd0e32f9bf6b28>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in <c9f8153c41de4f8cbafd0e32f9bf6b28>:0 
  at System.Runtime.CompilerServices.ConfiguredTaskAwaitable+ConfiguredTaskAwaiter.GetResult () [0x00000] in <c9f8153c41de4f8cbafd0e32f9bf6b28>:0 
  at OmniSharp.Extensions.LanguageServer.LspRequestRouter+<RouteRequest>d__11.MoveNext () [0x00161] in <43976da1155b4e5492c57f3d85e24f0e>:0 
```

**Comment by bqv** [comment]
duplicate #1742

**Comment by filipw** [comment]
fixed with #2005
